src/core/jpx_stream.js

```diff
@@ -42,13 +42,25 @@ class JpxStream extends DecodeStream {
   }
 
   readBlock(ignoreColorSpace) {
+    return this.decodeImage(null, ignoreColorSpace);
```

Comment:
Why the `return` here, since the `readBlock` method isn't expected to return any data?